.github/workflows/test_pyopenms.yml

```diff
@@ -0,0 +1,35 @@
+name: Test nightly pyOpenMS on different platforms
+
+on:
+  workflow_dispatch: # Trigger manually
+  workflow_run:
+    workflows: [ "pyopenms-wheels" ]
+    types:
+      - completed
+
+jobs:
+  test:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Cool. But this will not work, since our wheels are not standardized by versions.
See #6704

Reply:
If you check the pyopenms-wheels job, you will see that linux builds different versions for example.

Reply:
And it is not very maintainable to have multiple places in which to change versions.

Reply:
So what should we do, test one Python version, e.g. the latest we build for?

Reply:
No. Introduce a file with list of python versions to build. That is used by all CI. See #6704

Reply:
See e.g. https://www.cynkra.com/blog/2020-12-23-dynamic-gha/ how to create a GHA matrix dynamically.

Reply:
The pyopenms-wheels scripts can just read it in shell